Directory.Packages.props

```diff
@@ -21,9 +21,12 @@
     <PackageVersion Include="MinVer" Version="5.0.0" />
     <PackageVersion Include="Moq" Version="4.18.4" />
     <PackageVersion Include="morelinq" Version="4.2.0" />
+    <PackageVersion Include="MSBuild.StructuredLogger" Version="2.2.317" />
     <PackageVersion Include="MSTest.TestFramework" Version="3.5.1" />
     <PackageVersion Include="MSTest.Analyzers" Version="3.5.1" />
     <PackageVersion Include="MSTest.TestAdapter" Version="3.5.1" />
+    <PackageVersion Include="Microsoft.Build.Framework" Version="17.5.0" />
+    <PackageVersion Include="Microsoft.Build.Locator" Version="1.6.1" />
```

Comment:
I have a concern here. Are you running as a standalone .NET 6 application currently? If so, you won't be able to load MSBuild from new SDKs. Have you tried on a machine that has _only_ .NET 8 installed?

Reply:
This seems to work in the unit tests which should only have .NET6 installed (it's installing [here](https://github.com/microsoft/component-detection/actions/runs/10950179321/workflow?pr=1250#L29) and if I understand that action correctly, it uses `global.json` to determine what to install, so no .NET 8 on the CI machine)

Reply:
Right, the problem will be when you run the tool on a different machine which has only .NET 8 installed.